_component_code/text-fields.md

```diff
@@ -2,7 +2,7 @@
 title: Text Fields
 layout: code
 category: Components Code
-code: components/text-inputs
+code: components/text-input
```

Comment:
So... we split `components/text-inputs` into `components/text-input` and `components/textarea`. Will that be a problem?

Reply:
mainly it just means we'll only show one half of the examples. We *could* do some fancy template messing and get both on here, but I'm inclined to leave it for now as we're almost to Thunderdome.